src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/entities/DocumentTag.java

```diff
@@ -51,6 +51,7 @@ public enum DocumentTag {
     INTERNAL_DECIDE_AN_APPLICATION_LETTER("internalDecideAnApplicationLetter"),
     INTERNAL_DET_MARK_AS_ADA_LETTER("internalDetMarkAsAdaLetter"),
     INTERNAL_DETAINED_EDIT_CASE_LISTING_LETTER("internalDetainedEditCaseListingLetter"),
+    INTERNAL_FTPA_SUBMITTED_LETTER("internalFtpaSubmittedLetter"),
```

Comment:
Would it not be better to have tag as "internalFtpaSubmittedAppellantLetter", as there can be an "internalFtpaSubmittedRespondentLetter" too which may cause confusion.